website/integrations/services/gitea/index.md

```diff
@@ -62,14 +62,93 @@ Change the following fields
 -   OAuth2 Provider: OpenID Connect
 -   Client ID (Key): Step 1
 -   Client Secret: Step 1
--   Icon URL: https://goauthentik.io/img/icon.png
+-   Icon URL: https://authentik.company/static/dist/assets/icons/icon.svg
 -   OpenID Connect Auto Discovery URL: https://authentik.company/application/o/gitea-slug/.well-known/openid-configuration
 -   Additional Scopes: `email profile`
 
 ![](./gitea1.png)
 
 `Add Authentication Source` and you should be done. Your Gitea login page should now have a `Sign in With` followed by the authentik logo which you can click on to sign-in to Gitea with Authentik creds.
 
+### Step 4 _(optional Claims for authorization management)_
+
+:::note
+This step is **optional** and shows how to set claims to control the permissions of users in gitea by adding them to groups.
+:::
+
+#### Define Groups
+
+The following groups will be used:
+
+-   `gituser` for normal Gitea users.
+-   `gitadmin` for Gitea users with administrative permissions.
+-   `gitrestricted` for restricted Gitea users.
+
+:::note
+Users who are in none of these groups will not be able to log in to gitea.
+:::
+
+In authentik, create three groups (under _Directory/Groups_) with the _Name_ as mentioned above and leave other settings untouched.
+
+:::note
+You can add Members to the groups now or anytime later.
+:::
+
+#### Create Custom Property Mapping
+
+In authentik, create a custom property mapping (under _Customization/Property Mappings_) which has the type **Scope Mapping**.
+
+:::note
+Only settings that have been modified from default have been listed.
+:::
+
+-   Name: authentik gitea OAuth Mapping: OpenID 'gitea'
+-   Scope name: gitea
+
+And as **Expression** set the following:
+
+```(python)
+gitea_claims = {}
+if any(group.name == "gituser" for group in request.user.ak_groups.all()):
```

Comment:
```suggestion
if request.user.ak_groups.filter(name="gituser").exists():
```

applies for all three checks, the current policy would fetch all groups 3 times and then check in python versus checking at database level

Reply:
done in https://github.com/goauthentik/authentik/pull/3946/commits/3f6d4b0eec37b092a3620ffabbef8b33377dd23c